earth_enterprise/src/third_party/skia/SConscript

```diff
@@ -147,9 +175,6 @@ sgl_opts_ssse3_objects = map(
 sgl_opts_files = map(lambda f: os.path.join(sgl_src_abs_dir, f), sgl_opts_cpps)
 sgl_opts_ssse3_files = map(lambda f: os.path.join(sgl_src_abs_dir, f), sgl_opts_ssse3_cpps)
 
-skia_source = File('#/../../earth_enterprise/third_party/skia/%s.tar.gz'
-                   % skia_version).abspath
-
 skia_source_files = sgl_opts_files + sgl_opts_ssse3_files + sgl_cpps
 skia_target = '%s/.extract_skia' % current_dir
```

Comment:
wonder if we need this anymore...  might not hurt but might not need it too.

Reply:
I think this is still important in order to keep the ordering of the build steps correct. The library build still must wait for the tarball to be extracted.

Reply:
Scons will wait because it now knows that the source files to the build command is created by this untar command.  If you give scons all the correct information it does the right thing.  When you take short cuts and hide information from it then it will sometimes do the wrong thing because it doesn't know the full details.  But leaving the dot file for now isn't hurting anything.